src/main/java/org/zalando/nakadi/domain/EventTypeBase.java

```diff
@@ -201,22 +189,6 @@ public void setOptions(final EventTypeOptions options) {
         this.options = options;
     }
 
-    public Set<String> getWriteScopes() {
-        return Collections.unmodifiableSet(writeScopes);
-    }
-
-    public void setWriteScopes(final Set<String> writeScopes) {
```

Comment:
what will happen if somebody will try to create ET having this property? Will request fail or will Nakadi just ignore this property?